whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmServer.java

```diff
@@ -50,33 +52,39 @@ public void start() {
             return;
         }
 
-        serverChannel = bootstrapChannel(nrtmServerPipelineFactory, nrtmPort, "NRTM DUMMIFIER");
-        port = ((InetSocketAddress) serverChannel.getLocalAddress()).getPort();
+        serverChannel = bootstrapChannel(nrtmServerChannelInitializer, nrtmPort);
+        port = ((InetSocketAddress) serverChannel.localAddress()).getPort();
+        LOGGER.info("NRTM server listening on port {}", port);
     }
 
-    private Channel bootstrapChannel(final ChannelPipelineFactory serverPipelineFactory, final int port, final String instanceName) {
-        channelFactory = new NioServerSocketChannelFactory();
-        final ServerBootstrap bootstrap = new ServerBootstrap(channelFactory);
-        bootstrap.setPipelineFactory(serverPipelineFactory);
-        bootstrap.setOption("backlog", 200);
-        // apply TCP options to accepted Channels. Ref. https://netty.io/3.10/guide/
-        bootstrap.setOption("child.tcpNoDelay", true);
-        bootstrap.setOption("child.keepAlive", true);
-
-        final Channel channel = bootstrap.bind(new InetSocketAddress(port));
-        final int actualPort = ((InetSocketAddress) channel.getLocalAddress()).getPort();
-        LOGGER.info("NRTM server listening on port {} ({})", actualPort, instanceName);
-        return channel;
+    // TODO: [ES] channel was not returned?
```

Comment:
@eshryane it is returned from the private `bootstrapChannel` method, but not from the `start` method. It is exactly the way it has been, as can be seen in the current version with Netty3 [here](https://github.com/RIPE-NCC/whois/blob/a4650eb2d3ea9e5dee75a8aa70602b9ba7f921ce/whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmServer.java#L57)

Reply:
I fixed the method signature in 651de2 to return a `Channel` to match master, and added a comment for review.

I now removed the TODO to match the method.